### DISCUSSION TITLE
{Icon} {Label} component

### DISCUSSION THREAD

**Discussion by gitdallas** [patternfly/patternfly#5813] [Feature requests]
Pretty common to have an Icon with a Label.

Red Hat Insights has an example here using relative top spacing to line things up: https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/src/Battery/Battery.tsx#L65

Konveyor UI has an example here using flex layout: https://github.com/konveyor/tackle2-ui/pull/1250/files#diff-60b1fb4083b0e006fbe9eb403ed4c7a33d79e11e37bd8f211caef06b8cb2d84fR94

I'm wondering if PF could either have a component to encompass this in a best practice or at least have some sort of guideline/demo with best practice.

@mcoker mentioned pendo might help with something like this?

**Comment by mcoker** [comment]
I'm not sure about pendo 😅, maybe you meant penta? This is a really common pattern that we use within patternfly components, and I see in the wild in products as well. I'd love to see spacer tokens for this sort of thing or possibly support in a component like the icon component.
